### PR TITLE
Math magicians: components

### PR DESCRIPTION
**Pull request by Nkaleth**
Objectives of this PR:

- Delete all the boilerplate from Create React App (text, images, styles).
- Create a new directory called components.
- Inside components, create a new Calculator.js file.
- In Calculator.js, create a React component that matches this design: Calculator
- Implement the design only (JSX and CSS)
- Develop just the UI of the calculator; the math logic to make it work will be added in the next project.
- Use functional components
- Create at least 1 child component
- Once the Calculator is ready, import it and use it in your top-level component: App.js
- Test it in the browser; it should render without problems.

### CODE REVIEW

File: src/components/Calculator.css [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+.ButtonsCont {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+.Numbers {
+  width: 100%;
+  display: grid;
```
Comment (thecodechaser) [inline src/components/Calculator.css]: - [x] Please make sure your design matches the given design and fix the below issues.
  - Please remove the gap between the digit buttons and the operational buttons.
  - Please remove the colored border from the operational buttons and use the color `#F5913E` for the background.


### Your design

image

### Given design

image

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+          <button className="ButtonClassOper" type="button">÷</button>
+          <button className="ButtonClassOper" type="button">x</button>
+          <button className="ButtonClassOper" type="button">-</button>
+          <button className="ButtonClassOper" type="button">+</button>
+          <button className="ButtonClassOper" type="button">=</button>
+        </section>
+      </section>
+    </div>
+  );
+}
```
Comment (thecodechaser) [inline src/components/Calculator.js]: - [ ] Please use at least 1 child component as requested in the project requirements. I would suggest creating a different component for buttons and calling on this component.

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @Nkaleth👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- Highly organized code. ✔️
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by GzimAsani — APPROVED** [review]
# STATUS: APPROVED :100:         
## There is nothing else to say than it's time to merge... :partying_face: 
Cheers and Happy coding! :clap:  :computer:  :wine_glass: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


## Optional suggestions


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

### Please be reminded to not open a new pull request after you get a review, only use the same one :thought_balloon: 


 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**